api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/service/v1/events/schedulers/EventTaskSchedulerAsyncService.java

```diff
@@ -237,11 +244,16 @@ public void findModifiedSchoolsAndUpdateSdcSchoolCollection() {
       return;
     }
 
+    Optional<CollectionTypeCodeEntity> optionalCollectionTypeCodeEntity = this.collectionTypeCodeRepository.findByCollectionTypeCode(activeCollection.getCollectionTypeCode());
+    CollectionTypeCodeEntity collectionTypeCodeEntity = optionalCollectionTypeCodeEntity.orElseThrow(() -> new EntityNotFoundException(CollectionTypeCodeEntity.class, "collectionTypeCode", activeCollection.getCollectionTypeCode()));
+    List<CollectionCodeCriteriaEntity> collectionCodeCriteria = this.collectionCodeCriteriaRepository.findAllByCollectionTypeCodeEntityEquals(collectionTypeCodeEntity);
+    final List<SchoolTombstone> schoolTombstonesFromCollectionCodeCriteria = closeCollectionService.getListOfSchoolIDsFromCriteria(collectionCodeCriteria);
+
     restUtils.populateSchoolMap();
     final List<SchoolTombstone> schoolTombstones = restUtils.getSchools();
```

Comment:
Do we need this?

Reply:
it was being used for findClosedSchoolsAndDeleteSdcCollection - but we might actually want to use the collection code criteria here as well

Reply:
switched findClosedSchoolsAndDeleteSdcSchoolCollection to getListOfSchoolIDsFromCriteria